packages/galeri/src-xpetra/Galeri_HelmholtzFEM2DProblem.hpp

```diff
@@ -61,12 +61,12 @@ namespace Galeri {
 
   namespace Xpetra {
 
-    template <typename Scalar, typename LocalOrdinal, typename GlobalOrdinal, typename Map, typename Matrix, typename MultiVector>
-    class HelmholtzFEM2DProblem : public Problem_Helmholtz<Map,Matrix,MultiVector> {
+    template <typename Scalar, typename LocalOrdinal, typename GlobalOrdinal, typename Map, typename Matrix, typename MultiVector, typename RealValuedMultiVector>
```

Comment:
Why does HelmholtzFEM2DProblem still take a RealValuedMultiVector template parameter?

Reply:
Good catch, this only gets compiled with `MueLu_ENABLE_Experimental=ON`, I'll take care of this.